config/default/rdfImporterConfig.conf.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+return new oat\oatbox\config\ConfigurationService(array(
+    'strategy' => 'fail'
```

Comment:
I would use the constants here instead:

```suggestion
    RdfImporter::OPTION_STRATEGY => RdfImporter::OPTION_STRATEGY_FAIL_ON_DUPLICATE
```